Add missing steps to container creation (#122918)
Azure/azure-content: articles/active-directory-b2c/customize-ui-with-html.md

To create a public container in Blob storage, perform the following steps:

> 1. Under **Settings** in the left-hand menu, select **Configuration**.

Reviewer: ```suggestion
1. Under **Settings** in the leftmost menu, select **Configuration**.
```

> 1. Enable **Allow Blob anonymous access**

Reviewer: ```suggestion
1. Enable **Allow Blob anonymous access**.
```

> 1. Select **Save**

Reviewer: ```suggestion
1. Select **Save**.
```
